databox/core/bin/databox-process.ts

```diff
@@ -0,0 +1,50 @@
+import IDataboxWrapper from "@ulixee/databox-interfaces/IDataboxWrapper";
+import { IMessage, IResponse } from "../interfaces/ILocalDataboxProcess";
+
+function sendToParent(response: IResponse): void {
+  process.send(response);
+}
+
+function exit(): void {
+  process.exit();
+}
+
+process.on('SIGINT', exit);
```

Comment:
Isn't this the same functionality as the process?

Reply:
What do you mean? It's listening for SIGINT (Ctrl-C) on this "process", which is the "child process".

Reply:
Are you meaning to refer to a different process? I thought you were using the global process variable. So you're listing for sighup and exit and global process and saying to exit. I must be missing something

Reply:
This is what we used for the webvm in Ulixee 1.x. I kept it similar.

What specifically are you suggesting? That we don't need to run exit() on SIGINT? Or that we don't need to run exit on the `exit` event?

Reply:
Oh, weird. I just would have expected the process to already exit for all these interrupts. It seemed like unnecessary code.

Reply:
I think it is unnecessary for some of the signals, but not all of them. In Ulixee v1.x we were flushing logs in the exit method so we ran all the signals through it.